.github/workflows/rust.yml

```diff
@@ -60,3 +59,20 @@ jobs:
         with:
           command: install
           args: gitoxide
+
+  cargo-deny:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
I am loving this! There is so much I don't know about how GitHub actions can be used. This is definitely going into my toolbox :).